src/stratis_cli/_parser/_logical.py

```diff
@@ -55,7 +55,7 @@
                 (
                     "pool_name",
                     dict(action="store", default=None, nargs="?", help="Pool name"),
-                )
+                ),
```

Comment:
This looks like a stray edit, although I'm surprised that black doesn't either enforce or prohibit. Please see if you can remove it.